### PR TITLE
Airthings Integration docs - mods for api

### PR DESCRIPTION
**Pull request by BALSwim50W**
## Proposed change
Documentation changes to facilitate the Airthings connection in order to get the required id and secret required by the integration.

## Type of change

- [ ] Spelling, grammar or other readability improvements (`current` branch).
- [x] Adjusted missing or incorrect information in the current documentation (`current` branch).
- [ ] Added documentation for a new integration I'm adding to Home Assistant (`next` branch).
  - [ ] I've opened up a PR to add logo's and icons in [Brands repository](https://github.com/home-assistant/brands).
- [ ] Added documentation for a new feature I'm adding to Home Assistant (`next` branch).
- [ ] Removed stale or deprecated documentation.

## Additional information

- Link to parent pull request in the codebase: 
- Link to parent pull request in the Brands repository: 
- This PR fixes or closes issue: 

## Checklist

- [x] This PR uses the correct branch, based on one of the following:
  - I made a change to the existing documentation and used the `current` branch.
  - I made a change that is related to an upcoming version of Home Assistant and used the `next` branch.
- [x] The documentation follows the Home Assistant documentation [standards][].

[standards]: https://developers.home-assistant.io/docs/documenting/standards

### CODE REVIEW

**Comment by homeassistant** [comment]
Hi @BALSwim50W, 

It seems you haven't yet signed a CLA. Please do so [here](https://home-assistant.io/developers/cla_sign_start/?pr=home-assistant&#x2F;home-assistant.io#20531).

Once you do that we will be able to review and accept this pull request.

Thanks!

**Review by Danielhiversen — DISMISSED** [review]
Thanks. Great 👍

File: source/_integrations/airthings.markdown [hunk 1]
```diff
@@ -18,6 +18,32 @@ Integrates Airthings sensors into Home Assistant.
 
 [Airthings](https://www.airthings.com/) provide different sensors for measuring the air quality. The focus specially on Radon sensors.
 
-Login [here](https://dashboard.airthings.com/integrations/api-integration) to find your credentials.
+Requires Airthings hardware and a valid Airthings Dashboard login.
 
-{% include integrations/config_flow.md %}
+### Airthings API setup (needed to aquire the required ID and Secret for the Home Assistant Airthings Integration).
```
Comment (Danielhiversen) [inline source/_integrations/airthings.markdown]:
```suggestion
### Airthings API setup (needed to acquire the required ID and Secret for the Home Assistant Airthings Integration).
```

File: source/_integrations/airthings.markdown [hunk 2]
```diff
@@ -18,6 +18,32 @@ Integrates Airthings sensors into Home Assistant.
 
 [Airthings](https://www.airthings.com/) provide different sensors for measuring the air quality. The focus specially on Radon sensors.
 
-Login [here](https://dashboard.airthings.com/integrations/api-integration) to find your credentials.
+Requires Airthings hardware and a valid Airthings Dashboard login.
 
-{% include integrations/config_flow.md %}
+### Airthings API setup (needed to acquire the required ID and Secret for the Home Assistant Airthings Integration).
```
Comment (frenck) [inline source/_integrations/airthings.markdown]:
```suggestion
## Prerequisites

Airthings API setup (needed to acquire the required ID and Secret for the Home Assistant Airthings Integration).
```

File: source/_integrations/airthings.markdown [hunk 3]
```diff
@@ -18,6 +18,32 @@ Integrates Airthings sensors into Home Assistant.
...
+6. Select a Resource Scope
+
+7. Select an access Type ie. Confidential
+
+8. Select a flow type
+
+9. Set enable to "on"
+
+10. Save the settings
+
```
Comment (frenck) [inline source/_integrations/airthings.markdown]:
```suggestion
1. Login [here](https://dashboard.airthings.com/integrations/api-integration) to find your credentials.
2. Select [Integrations](https://dashboard.airthings.com/integrations/api-integration) from the left side-bar.
3. Click on "Request API Client" in order to set up an API connection.
4. Name your API connection ie. "Home Assistant Integration".
5. Give the connection an accurate description.
6. Select a Resource Scope.
7. Select an access Type, i.e., Confidential.
8. Select a flow type.
9. Set enable to "on".
10. Save the settings.

```

**Review by frenck — CHANGES_REQUESTED** [review]

**Review by frenck — APPROVED** [review]
Thanks, @BALSwim50W 👍

**Pull request merged**